test/images/resource-consumer/consume-cpu/BUILD

```diff
@@ -13,9 +13,50 @@ go_binary(
 
 go_library(
     name = "go_default_library",
-    srcs = ["consume_cpu.go"],
+    srcs = [
+        "consume_cpu.go",
+        "consume_cpu_windows.go",
+    ],
     importpath = "k8s.io/kubernetes/test/images/resource-consumer/consume-cpu",
-    deps = ["//vendor/bitbucket.org/bertimus9/systemstat:go_default_library"],
+    deps = select({
```

Comment:
I'm not a Bazel wizard, but is there a way to make deps select `golang.org/x/sys/windows:go_default_library` only for the windows platform, and the `bertimus9...` one for everything else? Maybe @ixdy knows how...?

Reply:
I think that's what this is doing already. (also, I'm pretty sure this section was auto-generated by gazelle.)

Reply:
I'm not sure why gazelle decided to elaborate all platforms instead of using the `//conditions:default` case, but alas, that's what it did.

Reply:
Indeed, this was auto-generated, the bazel job was complaining about it, if it wasn't like this. I'm not a fan of how this looks either.

Reply:
Ah OK. I was just wondering if there was a less verbose way of doing the same thing. But as it is autogenerated, I think there's no need to discuss further